yarn.lock

```diff
@@ -7651,6 +7651,17 @@ globby@13.1.4, globby@^13.1.1, globby@^13.1.4:
     merge2 "^1.4.1"
     slash "^4.0.0"
 
+globby@^13.1.1, globby@^13.2.0:
```

Comment:
Should wait for `cpy-cli` bump to avoid introducing yet another `globby` version. 🙈